src/main/java/uk/gov/hmcts/reform/civil/service/docmosis/DocmosisTemplates.java

```diff
@@ -2,6 +2,7 @@
 
 public enum DocmosisTemplates {
     N1("CV-UNS-CLM-ENG-00727.docx", "sealed_claim_form_%s.pdf"),
+    N1_MULTIPARTY_SAME_SOL("CV-UNS-CLM-ENG-00762.docx", "sealed_claim_form_%s.pdf"),
```

Comment:
Please suggest a better name for this if you feel it needs it, I wasn't sure what `N1` stood for